src/_gettsim/parameters/ges_rente.yaml

```diff
@@ -1727,75 +1962,203 @@ altersgrenze_langj_versicherte_vorzeitig:
   note: https://www.sozialgesetzbuch-sgb.de/sgbvi/236.html
   1980-01-01:
     scalar: 63.0
+  1989-12-18:
+    reference: Rentenreformgesetz 1992. BGBl. I S. 2261 1989 § 41
+    note: Early retirement age descreased from 63 to 62 starting from birth cohort 1944.
+    max_birthyear_old_regime: 1943
+    min_birthyear_new_regime: 1944
+    entry_age_old_regime: 63.0
+    entry_age_new_regime: 62.0
+  1996-09-27:
+    reference: Wachstums- und Beschäftigungsförderungsgesetz 1996. BGBl. I S. 1461 1996
+    note: ERA back to 63.
+    scalar: 63.0
+  1997-12-16:
+    reference: Rentenreformgesetz 1999. BGBl. I S. 2998 1997
+    note: ERA back to 62.
+    scalar: 62.0
+  2007-04-20:
+    reference: RV-Nachhaltigkeitsgesetz 2004. BGBl. I S. 1791 2004
+    note: ERA back to 63.
+    scalar: 63.0
```

Comment:
Note (more to myself than to others, but feel free to chime in if you know more): The ERA reforms for the Rente f. langj. Versicherte seem very strange as it jumps back and forth from 63 to 62.